resources/assets/coffee/react/profile-page/extra.coffee

```diff
@@ -90,20 +126,51 @@ class ProfilePage.Extra extends React.Component
 
   _removeListeners: ->
     $.unsubscribe '.profileContentsExtra'
+    $.unsubscribe 'profilePageExtraToggle:toggleDragging'
```

Comment:
if you suffixed the name with `.profileContentsExtra`, you don't need to add `$.unsubscribe` here.